stl/inc/mutex

```diff
@@ -125,41 +125,42 @@ _INLINE_VAR constexpr try_to_lock_t try_to_lock{};
 
 // CLASS TEMPLATE unique_lock
 template <class _Mutex>
-class _NODISCARD unique_lock { // whizzy class with destructor that unlocks mutex
+class unique_lock { // whizzy class with destructor that unlocks mutex
 public:
     using mutex_type = _Mutex;
 
     // CONSTRUCT, ASSIGN, AND DESTROY
     unique_lock() noexcept : _Pmtx(nullptr), _Owns(false) {}

```

Comment:
Note that I haven't annotated the default and `defer_lock_t` constructors, since discarding a non-owning lock is not a bug.